app/controllers/authenticated_controller.rb

```diff
@@ -10,6 +12,19 @@ def require_authentication
     redirect_to "/auth/openid_connect"
   end
 
+  def require_registration
+    return if Rails.env.test?
```

Comment:
why?

Reply:
registration depends on a SCP integration to be stubbed out which is complexity I've avoided for migration/have relied on manual testing locally

Reply:
I've added a story to validate all of this with a request spec once we've finished/have gone live.https://transformuk.atlassian.net/browse/HMRC-1059